### PR TITLE
OV-138: Add script to timeline component

### PR DESCRIPTION
**Pull request by Alblupynos**
Scripts properly added to the timeline on store changes.
image

### CODE REVIEW

File: frontend/src/bundles/studio/components/timeline/timeline.tsx [hunk 1]
```diff
@@ -136,6 +142,16 @@ const Timeline: React.FC<Properties> = ({ initialRange, initialItems }) => {
...
+    const updatedScripts = useAppSelector(({ studio }) => studio.scripts);
+    useEffect(() => {
+        setItems((previous) =>
+            setItemsSpan({
+                ...previous,
+                script: getScriptItems(updatedScripts),
+            }),
+        );
+    }, [updatedScripts]);
+
```
Comment (o-nedashkivska) [inline frontend/src/bundles/studio/components/timeline/timeline.tsx:154]: I think we should remove mocked items and stop passing it to TimelineView. We should read updatedScripts from store in ScriptsRow component. And in handlers like onDragEnd etc dispatch action to change scripts in store

Comment (o-nedashkivska) [reply frontend/src/bundles/studio/components/timeline/timeline.tsx:154]: Because right now reorder scripts with dnd it will not get to store

Comment (nikita-remeslov) [reply frontend/src/bundles/studio/components/timeline/timeline.tsx:154]: Looks like avatar mocks better to remove with avatar ticket

Comment (o-nedashkivska) [reply frontend/src/bundles/studio/components/timeline/timeline.tsx:154]: Yes, agree
I will do it in this ticket https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/165

File: frontend/src/bundles/studio/components/timeline/timeline.tsx [hunk 2]
```diff
@@ -49,18 +33,32 @@ const Timeline: React.FC<Properties> = ({
 
             const updatedSpan = activeItem.getSpanFromResizeEvent?.(event);
 
-            if (!updatedSpan || activeItemType === RowNames.SCRIPT) {
```
Comment (o-nedashkivska) [inline frontend/src/bundles/studio/components/timeline/timeline.tsx:52]: Please, leave this condition and remove `resizeScript` reducer. Scripts must not be resizable.

Comment (Alblupynos) [reply frontend/src/bundles/studio/components/timeline/timeline.tsx]: Done)

**Review by o-nedashkivska — APPROVED** [review]

**Pull request merged**